Sources/SwiftGraphQLClient/Client/Operation.swift

```diff
@@ -105,6 +105,9 @@ public enum CombinedError: Error {
     
     /// An error occured and it's not clear why.
     case unknown(Error)
+
+    /// Combines different kinds of errors into one
+    indirect case combined([CombinedError])
```

Comment:
How about we extend the `parsing` case with an array of `[GraphQLError]`? My understanding is that only those two ever collide. Did I miss anything?

Reply:
Probably you're right - for now only they collide. But this way we will kinda make them all look like they are parsing errors, while at least some of them not. This new case shows that there might be mixed types of errors

Reply:
Prefer not to, to avoid making all of them look like parse errors